docs/endpoints/multi_user.rst

```diff
@@ -0,0 +1,909 @@
+Multi-User Compute Endpoints
+****************************
+
+Multi-user Compute endpoints (MEP) enable administrators to securely offer compute
+resources to users without mandating the typical shell access (i.e., ssh).  The basic
+thrust is that the administrator of a MEP creates a configuration template, which is
+then populated by user data and passed to a user endpoint process, or UEP.  The UEP runs
+as the local POSIX user, with the only difference from a standard :abbr:`CEP
+([single-user] Compute endpoint)` being that it was started not by a human at the
+command line but on demand by the parent MEP.  In this manner, administrators may preset
+endpoint configuration options (e.g., Torque, PBS, Slurm) and offer user-configurable
+items (e.g., account id, problem-specific number of blocks), while at the same time
+lowering the barrier to use of the resource.
+
+From an SDK user's perspective, the change to use a MEP is minimal, requiring still the
+endpoint agent identifier, and optionally any template-variable values.  An example
+SDK interaction might look like:
+
+.. code-block:: python
+   :emphasize-lines: 7, 9
+
+   from globus_compute_sdk import Executor
+
+   def hello_from_node(name: str):
+       import os
+       return f"Hello, {name}; I ran on {os.uname().nodename}"
+
+   mep_id = "id_as_shared_by_mep_admin"  # perhaps via email, or a web site
+   with Executor() as ex:
+       ex.endpoint_id = mep_id  # no change from a CEP interaction
+       f = ex.submit(hello_from_node, "Obi-Wan")
+       print(f.result())
+
+This interaction is no different than for use of a CEP (in fact, the Tutorial Endpoint
+is a MEP, and this interaction works there as well!), although does not customize the
+configuration.  For that, the only necessary addition is the ``user_endpoint_config``:
+
+.. code-block:: python
+   :emphasize-lines: 10
+
+   from globus_compute_sdk import Executor
+
+   def hello_from_node(name: str):
+       import os
+       return f"Hello, {name}; I ran on {os.uname().nodename}"
+
+   mep_id = "id_as_shared_by_mep_admin"  # perhaps via email, or a web site
+   with Executor() as ex:
+       ex.endpoint_id = mep_id  # no change from a CEP interaction
+       ex.user_endpoint_config = {"ACCOUNT_ID": "OBIWAN123"}
+       f = ex.submit(hello_from_node, "Obi-Wan")
+       print(f.result())
+
+More on the specifics later.  The rest of this section is oriented toward cluster
+administrators, and assumes a passable understanding of installation and use of a
+single-user Compute endpoint.
+
+.. tip::
+
+   For those just looking for the instructions, see the `Administrator Quickstart`_,
+   below.
+
+.. _repo-based-installation:
+
+Repository-Based Installation
+=============================
+
+The ``globus-compute-endpoint`` project is available on PyPI, and is also available in
```

Comment:
Maybe add a `pip install` example for the skimming reader.

Reply:
I'm not terribly concerned about that point.  This page is pretty explicitly oriented toward administrators, and we even explicitly say just above this:

> The rest of this section is oriented toward cluster administrators, and assumes a passable understanding of installation and use of a single-user Compute endpoint.

I hear the skimming point, but that's also not the focus of this section.